features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -21,6 +21,30 @@ export function SidebarNavigation() {
   const { isSidebarCollapsed, toggleSidebar } = useContext(NavigationContext);
   const [isMobileMenuOpen, setMobileMenuOpen] = useState(false);
 
+  //State for if it's mobile view or not and accounts for the server side rendering delay
+  const [isMobileView, setIsMobileView] = useState(
+    typeof window !== "undefined" ? window.innerWidth <= 1023 : false,
+  );
+
+  useEffect(() => {
+    const handleWindowResize = () => {
+      setIsMobileView(window.innerWidth <= 1023);
+    };
+
+    if (typeof window !== "undefined") {
+      window.addEventListener("resize", handleWindowResize);
+
+      //Cleans event listener
+      return () => {
+        window.removeEventListener("resize", handleWindowResize);
+      };
+    }
```

Comment:
The `useEffect` only runs in the browser so it should be ok to remove the `typeof window !== 'undefined` check